lib/psulib_traject/processors/call_number/dewey.rb

```diff
@@ -15,7 +47,32 @@ def scheme
     end
 
     def reduce
-      call_number
+      value = remove_by_regex
+      value = value[0...(value.index(LOOSE_MONTHS_REGEX) || value.length)] # remove loose months
+
+      if serial
+        self.class.remove_years(value)
+      else
+        value.strip
+      end
     end
+
+    private
+
+      # @note These are the original regex patterns from Stanford.
+      def remove_by_regex
+        case removables
+        when VOL_PATTERN
+          call_number.slice(0...call_number.index(removables[VOL_PATTERN])).strip
+        when VOL_PATTERN_LOOSER
```

Comment:
Code Climate is barking about this branch of our case statement. We don't have a test for it, and I feel like we ought to notify ourselves because we don't know if any of our records are actually hitting this or not. We could put a logger message, better yet, raise an exception. If we find that one of our records _is_ actually hitting this, we can put it into the test.

Reply:
I ran the full index on my local with raise an exception for the VOL_PATTERN_LOOSER branch and it did not hit it so I commented that out as we have in LC.